src/helpers/buildFilename.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @typedef {object} BuildPaperNameParam
+ * @property {string} qualificationName - Name of the paper qualification
+ * @property {string} [pageName] - Name of page (eg Front)
+ * @property {string} [fullname] - Fullname of contact
+ * @property {string} [formatedDate] - Date already formated
+ */
+
+/**
+ * Builded Paper name with qualification name & without use filename original
+ * @param {BuildPaperNameParam} opts
+ * @returns {string} Paper name with PDF extension
+ */
+export const buildFilename = opts => {
+  /*
+    Calling the stack's file creation method would trigger a ``status: "422", title: "Invalid Parameter"error if filename contains/`.
```

Comment:
Seems like there are some wrongly placed backquotes
```suggestion
    Calling the stack's file creation method would trigger a `status: "422", title: "Invalid Parameter"` error if filename contains `/`.
```